### PR TITLE
Build script update

### PR DESCRIPTION
**Pull request by yojagad**
- Generate artifacts for dev branch for perf tests

- Update build.ps1 for handling workers.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -268,6 +269,7 @@ function cleanExtension([string] $bitness) {
     $keepRuntimes = @('win', 'win-x86', 'win10-x86', 'win-x64', 'win10-x64')
     Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes -ErrorAction SilentlyContinue |
         Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
+    deleteDuplicateWorkers
```
Comment (pragnagopa) [inline build.ps1]: Undo this. Move it back to line 217. This needs to be called only once after both 32bit and 64bit folders are cleaned

File: build/initialize-pipeline.ps1 [hunk 2]
```diff
@@ -4,9 +4,11 @@ $bypassPackaging = $true
 $includeSuffix = $true
 Write-Host "SourceBranch: $sourceBranch, Build reason: $buildReason"
 
-if($sourceBranch.endsWith('master') -and ($buildReason -ne "PullRequest"))
+if(($sourceBranch.endsWith('master') -or $sourceBranch.endsWith('dev')) -and ($buildReason -ne "PullRequest"))
 {
-  $includeSuffix = $false
+  if($sourceBranch.endsWith('master')) {
```
Comment (alrod) [inline build/initialize-pipeline.ps1:9]: why do you need this if?

Comment (alrod) [inline build/initialize-pipeline.ps1:9]: Why do we include the suffix only for master? Is the suffix runtime version?

Comment (yojagad) [reply build/initialize-pipeline.ps1:9]: suffix is whether to include "-prerelease" string or not.

**Review by alrod — APPROVED** [review]

**Review by pragnagopa — APPROVED** [review]
Workers changes look good. Please do verify site extension artifact.

**Pull request merged**